.github/workflows/build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - uses: snok/install-poetry@v1.3
         with:
-          version: 1.5.1  # For python 3.7 compatibility
+          version: 1.6.1
```

Comment:
Done: 425c7a7941e93fe42f93ffbf27f96615dc3b5bf9

Reply:
Poetry failed to install on PyPy after this change. Probably transient.

Reply:
Yeah, that's strange, the same version (1.6.1) succeeded [here](https://github.com/requests-cache/requests-cache/actions/runs/6414458058/job/17414885597). I cleared the CI caches and it failed again, though. Let me see if I can reproduce it locally.

Reply:
It looks like sometime within the last 15 hours, GitHub Actions updated its version of the pypy3.9 interpreter ([released last week](https://www.pypy.org/posts/2023/09/pypy-v7313-release.html)).
Previous: `PyPy 7.3.12 with Python (3.9.17)`  (working)
Current: ` PyPy 7.3.13 with Python (3.9.18)`  (not working)

I tried running these actions locally with [act](https://github.com/nektos/act), but didn't get any more useful info yet. Unfortunately there's no way to specify a patch version of an interpreter in GitHub Actions, and this may be a pain to debug.

I'd appreciate help looking into it if you feel like it, but since this doesn't have anything to do with dropping python 3.7, I would also be fine with disabling pypy tests for now and creating a separate issue for it.

Reply:
I installed `pypy3.9-7.3.13` locally with pyenv and it failed too when trying to install using the official Poetry installer. These are the contents of the error log:

```
ModuleNotFoundError: No module named 'encodings'
debug: OperationError:
debug:  operror-type: ModuleNotFoundError
debug:  operror-value: No module named 'encodings'

Traceback:

  File "<stdin>", line 923, in main
  File "<stdin>", line 562, in run
```

I'll keep investigating.

Reply:
Using `pypy3.9-7.3.11` seems to fix the issue: https://github.com/requests-cache/requests-cache/actions/runs/6425264996/job/17447532241?pr=881

Reply:
Oh great! I wasn't aware of that syntax.